templates/acs.yaml

```diff
@@ -145,6 +154,17 @@ Parameters:
       Type: String
       Description: The Database password for Alfresco
       NoEcho: True
+    MQEndpoint:
+      Type: String
+      Description: MQ Endpoint for AmazonMQ connection
+    MQUsername:
+      Default: admin
+      Description: User name for the AmazonMQ
+      Type: String
+    MQPassword:
```

Comment:
The definition of these parameters should be the same as when defined in acs-deployment-master.yaml